src/modules/articles.js

```diff
@@ -1,8 +1,17 @@
 import axios from "axios";
 
-const getArticles = async () => {
-  let result = await axios.get("/articles");
-  return result.data.articles;
+const Articles = {
+  async index() {
+    let result = await axios.get("/articles");
+    return result.data.articles;
+  },
+  async show(articleId) {
+    try {
+      let result = await axios.get(`/articles/${articleId}`);
+      return result.data.article;
+    } catch (error) {
+      return error.response.data.error_message;
```

Comment:
Have you checked this out ugh your api that you get the response in this key? I’m just asking because I’m curious

Reply:
Thanks to this comment, now it is!